client/cli/src/params/pruning_params.rs

```diff
@@ -46,6 +46,7 @@ impl PruningParams {
 			.as_ref()
 			.map(|s| match s.as_str() {
 				"archive" => Ok(PruningMode::ArchiveAll),
+				"canonical" => Ok(PruningMode::ArchiveCanonical),
```

Comment:
"canonical" is more of an internal concept, and not a user friendly term. Let's call it "final" or "finalized"

Reply:
Ok , Let's set it "final".